tests/test_routes.py

```diff
@@ -8,38 +8,130 @@
 import os
 import logging
 from unittest import TestCase
-from unittest.mock import MagicMock, patch
+import random
 from service import app
-from service.models import db
+from service.models import db, init_db, Address, Customer
 from service.common import status  # HTTP Status Codes
+from tests.factories import AddressFactory, CustomerFactory
 
 
+DATABASE_URI = os.getenv(
+    "DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/testdb"
```

Comment:
docker-compose.yml has DATABASE_URI as "postgresql://postgres:postgres@postgres:5432/postgres"
Are we failing back to a testdb in case of failure of connection to postgres database?

Reply:
No, the second argument will be taken if for some reason our container doesnt have the DATABASE_URI env variable set.
I wanted to use postgres as the db only, but [test_models](https://github.com/CSCI-GA-2820-SP23-003/customers/blob/b790d6dcebacc23b4ddf762eb01d761bd1302952/tests/test_models.py#L14) had already used testdb, so used that to maintain consistency.

Reply:
Ideally we wont be ever falling back to testdb, but if you feel both test_models/routes should default to postgres as the db then I can change it.

Reply:
Okay, understood. 👍

Reply:
Let's discuss this during the call in 10 mins and merge then.